rulesets/resources/301validator.xml

```diff
@@ -13,7 +13,7 @@
       <sch:assert role='ERROR' test='h:SoftwareInfo/h:extension/h:IECCERICalculation/h:Version[text()="2024" or text()="2021" or text()="2018" or text()="2015"] or not(h:SoftwareInfo/h:extension/h:IECCERICalculation/h:Version)'>Expected SoftwareInfo/extension/IECCERICalculation/Version to be '2024' or '2021' or '2018' or '2015'</sch:assert>
       <sch:assert role='ERROR' test='count(h:SoftwareInfo/h:extension/h:EnergyStarCalculation/h:Version) &lt;= 1'>Expected 0 or 1 element(s) for xpath: SoftwareInfo/extension/EnergyStarCalculation/Version</sch:assert> <!-- Required if running ENERGY STAR calculation -->
-      <sch:assert role='ERROR' test='h:SoftwareInfo/h:extension/h:EnergyStarCalculation/h:Version[text()="SF_National_3.0" or text()="SF_National_3.1" or text()="SF_National_3.2" or text()="SF_Pacific_3.0" or text()="SF_Florida_3.1" or text()="SF_OregonWashington_3.2" or text()="MF_National_1.0" or text()="MF_National_1.1" or text()="MF_National_1.2" or text()="MF_OregonWashington_1.2"] or not(h:SoftwareInfo/h:extension/h:EnergyStarCalculation/h:Version)'>Expected SoftwareInfo/extension/EnergyStarCalculation/Version to be 'SF_National_3.0' or 'SF_National_3.1' or 'SF_National_3.2' or 'SF_Pacific_3.0' or 'SF_Florida_3.1' or 'SF_OregonWashington_3.2' or 'MF_National_1.0' or 'MF_National_1.1' or 'MF_National_1.2' or 'MF_OregonWashington_1.2'</sch:assert> <!-- See [EnergyStar=SFNational30] or [EnergyStar=SFNational31] or [EnergyStar=SFNational32] or [EnergyStar=SFPacific30] or [EnergyStar=SFFlorida31] or [EnergyStar=SFOregonWashington32] or [EnergyStar=MFNational10] or [EnergyStar=MFNational11] or [EnergyStar=MFNational12] or [EnergyStar=MFOregonWashington12] -->
+      <sch:assert role='ERROR' test='h:SoftwareInfo/h:extension/h:EnergyStarCalculation/h:Version[text()="SF_National_3.0" or text()="SF_National_3.1" or text()="SF_National_3.2" or text()="SF_National_3.3" or text()="SF_Pacific_3.0" or text()="SF_Florida_3.1" or text()="SF_OregonWashington_3.2" or text()="MF_National_1.0" or text()="MF_National_1.1" or text()="MF_National_1.2" or text()="MF_National_1.3" or text()="MF_OregonWashington_1.2"] or not(h:SoftwareInfo/h:extension/h:EnergyStarCalculation/h:Version)'>Expected SoftwareInfo/extension/EnergyStarCalculation/Version to be 'SF_National_3.0' or 'SF_National_3.1' or 'SF_National_3.2' or 'SF_Pacific_3.0' or 'SF_Florida_3.1' or 'SF_OregonWashington_3.2' or 'MF_National_1.0' or 'MF_National_1.1' or 'MF_National_1.2' or 'MF_OregonWashington_1.2'</sch:assert> <!-- See [EnergyStar=SFNational30] or [EnergyStar=SFNational31] or [EnergyStar=SFNational32] or [EnergyStar=SFPacific30] or [EnergyStar=SFFlorida31] or [EnergyStar=SFOregonWashington32] or [EnergyStar=MFNational10] or [EnergyStar=MFNational11] or [EnergyStar=MFNational12] or [EnergyStar=MFOregonWashington12] -->
```

Comment:
We also need to add logic below for, e.g., requiring the appropriate building type(s) for these versions. Maybe you can collapse the logic so we don't have to add a new `sch:pattern` each time we add new versions?